plagiarism/process.go

```diff
@@ -0,0 +1,136 @@
+package plagiarism
+
+import (
+    "io"
+    "io/ioutil"
+    "encoding/json"
+    "log"
+    "github.com/SpaceWhite/gomoss"
+    "fmt"
+    "os"
+)
+
+// GUIDsStruct use for decoding JSON
+type GUIDsStruct struct {
```

Comment:
Maybe call this `AnalysisRequest` since the client will send it to you as part as a request.